src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -6,10 +6,10 @@ To turn it on:
 1. Switch to Flutter's `main` channel:
 
    ```sh
-   flutter channel main
+   flutter channel main --no-cache-artifacts
    ```
 
-1. Make sure you have the latest Flutter SDK:
+1. Upgrade to the latest Flutter SDK and download artifacts:
 
    ```sh
    flutter upgrade
```

Comment:
Both `flutter channel main` and `flutter upgrade` download artifacts. If `flutter upgrade` upgrades your SDK, all the artifacts you downloaded for `flutter channel main` are thrown away. The `--no-cache-artifacts` skips that potentially throwaway work.